.github/CODEOWNERS

```diff
@@ -650,6 +650,7 @@ app/sidekiq/in_progress_form_cleaner.rb @department-of-veterans-affairs/vfs-auth
 app/sidekiq/kms_key_rotation @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/sidekiq/lighthouse @department-of-veterans-affairs/backend-review-group
 app/sidekiq/mhv @department-of-veterans-affairs/vsp-identity
```

Comment:
I would just update this to your team (instead of identity), then you don't need to update the individual files within the folder. As far as I can tell it looks like your team should own all the files within `sidekiq/mhv` anyways

Reply:
Thanks! Made both requested changes.